astro/src/components/docs/reference/ConfigurationOptions.astro

```diff
@@ -59,7 +59,7 @@ Display configuration values in format:
 
   <APIField name="database.password" type="String">
     <p>
-    The password that FusionAuth will use to connect to the database. If you are using [Silent Mode](/docs/get-started/download-and-install/silent-mode) or [Setup Wizard](/docs/get-started/run-in-the-cloud/setup-wizard), then this password might be used when the <code>database.user</code> is automatically created for you, depending on your database server and configuration options.
+    The password that FusionAuth will use to connect to the database. If you are using <a href="/docs/get-started/download-and-install/silent-mode">Silent Mode</a> or <a href="/docs/get-started/run-in-the-cloud/setup-wizard">Setup Wizard</a>, then this password might be used when the <code>database.user</code> is automatically created for you, depending on your database server and configuration options.
```

Comment:
Whey can't we use markdown links here? I don't think we want to be using literal HTML if at all possible.

Reply:
This is an astro component, so we're unable to use markdown links here.

Reply:
Can we write another astro component then so we don't have as much boiler plate HTML? Seems like we should rarely be writing HTML like this except in as part of a specific component.

Reply:
Filed https://github.com/FusionAuth/fusionauth-issues/issues/2697 to capture this work b/c it's a bigger refactoring.  (Astro complained with erroneous error messages on a first attempt at moving).